app/src/test/java/uk/ac/ebi/atlas/experimentpage/markergenes/JsonMarkerGenesControllerWIT.java

```diff
@@ -1,167 +1,169 @@
-//                .andExpect(jsonPath("$[0].cellGroupValueWhereMarker", isA(String.class)))
-//                .andExpect(jsonPath("$[0].cellGroupValue", isA(String.class)))
-//                .andExpect(jsonPath("$[0].x", isA(Number.class)))
-//                .andExpect(jsonPath("$[0].y", isA(Number.class)))
-//                .andExpect(jsonPath("$[0].geneName", isA(String.class)))
-//                .andExpect(jsonPath("$[0].value", isA(Number.class)))
-//                .andExpect(jsonPath("$[0].pValue", isA(Number.class)));
-//    }
-//
-//    @Test
-//    void isCellTypesMarkerGenesHeatmapDataIsValidJsonForCellTypeGroup() throws Exception {
-//        this.mockMvc
-//                .perform(get(cellTypesMarkerGenesHeatmapURL, "E-MTAB-5061")
-//                        .param("cellGroupType", "inferred cell type -ontology labels"))
-//                .andExpect(status().isOk())
-//                .andExpect(content().contentType(MediaType.APPLICATION_JSON_UTF8));
-//    }
-//
-//    @Test
-//    void invalidExperimentAccessionReturnsEmptyJson() throws Exception {
-//        this.mockMvc
-//                .perform(get(markerGeneCellTypeURL, "FOO")
-//                        .param("organismPart", "skin"))
-//                .andExpect(status().is2xxSuccessful());
-//    }
-//}
+package uk.ac.ebi.atlas.experimentpage.markergenes;
+
+import com.google.common.collect.ImmutableSet;
+import org.junit.Ignore;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.http.MediaType;
+import org.springframework.jdbc.datasource.init.ResourceDatabasePopulator;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.context.web.WebAppConfiguration;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+import uk.ac.ebi.atlas.configuration.TestConfig;
+import uk.ac.ebi.atlas.testutils.JdbcUtils;
+
+import javax.inject.Inject;
+import javax.sql.DataSource;
+
+import static org.hamcrest.CoreMatchers.isA;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+@Disabled
+@ExtendWith(SpringExtension.class)
+@WebAppConfiguration
+@ContextConfiguration(classes = TestConfig.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class JsonMarkerGenesControllerWIT {
```

Comment:
Please add a small comment here too for disabling.

Reply:
All the disabled tests are because we had failures in the previous Kubernetes. I added a task in our project to review `@Disabled` tests.